.kokoro/continuous/worker_0.cfg

```diff
@@ -0,0 +1,4 @@
+env_vars: {
+    key: "DJANGO_WORKER_INDEX"
+    value: 0
```

Comment:
Ah it looks like these need to be strings
```suggestion
    value: "0"
```

Reply:
Ack, thank you! Let me make that update!